src/cmd.ts

```
@@ -28,10 +29,16 @@ if ((!dir && !argv.version) || argv.help) {
     console.log('v' + require(__dirname + '/../package.json').version)
     process.exit(0)
 } else {
+    const baseDocumentsDir = path.resolve(dir)
+
     couchify({
-        baseDocumentsDir: dir,
+        baseDocumentsDir: baseDocumentsDir,
         id: argv.name
     }).then((result: DesignDocument) => {
+        try {
+            result.rewrites = require(baseDocumentsDir + '/rewrites.json')
+        } catch (er) { /* noop */ }
```

No silent failures. It's better yet to remove the try-catch.